tests/lib/mock_backend.py

```diff
@@ -104,19 +105,6 @@ class MockBackend(BaseBackend):
 # Create a default MockBackend instance for code which is still expecting this
 MockBackend = backend_factory(MssqlQueryEngine)
 
-# Generate an integer sequence to use as default IDs. Normally you'd rely on the DBMS to
-# provide these, but we need to support DBMSs like Spark which don't have this feature.
-next_id = iter(range(1, 2**63)).__next__
-
-
-# We need each NULL-able column to have an explicit default of NULL. Without this,
-# SQLAlchemy will just omit empty columns from the INSERT. That's fine for most DBMSs
-# but Spark needs every column in the table to be specified, even if it just has a NULL
-# value. Note: we have to use a callable returning `None` here because if we use `None`
-# directly SQLAlchemy interprets this is "there is no default".
-def null():
-    return None
-
```

Comment:
I think I ended up duplicating these for the generative tests. Can you use the extracted versions there as well?

Reply:
Done. I've also realised in doing this that they weren't using our special `null()` function, so I've added this in a separate commit. It's the thing we needed to add for the Spark tests and it's harmless for other databases so I figured we should use it by default.